activities/understanding-our-users/index.md

```diff
@@ -12,10 +12,14 @@ redirect_from:
 The work of the Foundation for Public Code impacts a wide set of stakeholders.
 This set of resources helps to understand our users, and how our work does or may impact them.
 
-## Existing pain Points
+## Existing pain points
 
 * Understanding the [pain points that public organizations](public-organizations.md) have whith their digital infrastructure
 
+## Existing enabling policies
+
+* Understanding the types of [enabling policies](enabling-policies.md) that can help public organizations and civil servants
```

Comment:
Missing trailing period.

Reply:
Yes, I was unsure what to do on this page. All other bullet points, except the last one under Additional resources, also miss it. Should it be added to all, only here, or removed on the single instance where it is?

Reply:
I agree, as it is a bullet point, it should be consistent with the others.